src/converter/data/dataConverter.ts

```diff
@@ -655,6 +661,29 @@ export abstract class DataConverter implements IConverter<IDataRepresentation> {
                 series.comparisonValue,
             );
 
+            // Second KPI Indicator
+            series.secondKPIIndicatorIndex = NumericValueUtils.isValueFinite(secondKPIIndicatorIndex)
+                ? secondKPIIndicatorIndex
+                : NaN;
+
+            if (isSecondKPIIndicatorIndexSpecified) {
+                series.points[0].kpiIndicatorIndexes.push(secondKPIIndicatorIndex);
+
+                if (series.settings.sparklineSettings.isActualVisible
+                    && series.settings.sparklineSettings.shouldActualUseKPIColors
+                    && series.points[0]
+                ) {
+                    const currentKPI: IKPIIndicatorSettings = settings
+                        .kpiIndicator
+                        .getCurrentKPI(secondKPIIndicatorIndex);
+
+                    const color: string = currentKPI && currentKPI.color
+                        || series.points[0].color;
+
+                    series.points[0].colors.push(color);
+                }
+            }
+
```

Comment:
I'm not entirely if I'm handling this part correctly, since the SecondComparisonValue wasn't shown in the SparkLine before this can most likely also be removed.

Reply:
The first part is actually needed, but I've removed the if clause as it would just conflict with the values of the first comparison value